Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -244,15 +244,23 @@ function xASL_wrp_ResampleASL(x)
     elseif x.modules.asl.bTimeEncoded

```

Comment:
Sorry that I only see this now, but `x.Q.NumberEchoTimes` is now only specified and used with `bTimeEncoded`, but this should also be used without Time encoded (i.e., for multiPLD or singlePLD). Does this always go correctly? Is this fourth dimension always echo time, for all type of Hadamard/multiPLD/singlePLD scans? Please generalize this.

Reply:
Yes. But where there's no demand, there's no offer. We don't have such a sequence and it would take a lot of time to implement and test something that is then never used...

Reply:
The rule is to generalize. If there is a special case then try to generalize. So we HAVE to come up with something that would work in all cases, that shouldn't be too difficult. E.g., (and please move this to #1181 ;)

Reason is: demand will come easily. By only creating new if-statements without generalization, code easily becomes convoluted. We need to simplify:

* `x.Q.numberEchoTimes` is always used and defaulted to 1
* Dimension 4 is always EchoTimes, Dimension 5 is always PLDs, Dimension 6 is always labeling duration (or some other order). These are all defaulted to 1

Reply:
ok. make a new issue for that.